cobigen-setup/index.asciidoc

```diff
@@ -0,0 +1,98 @@
+= A setup tutorial for devonfw Code-based incremental Generator(i.e. Cobigen)
+====
+This tutorial mainly focuses on the setup of CobiGen along with various commands to execute the generations.
+== Prerequisites

```

Comment:
Change '== Prerequisites' to '## Prerequisites'. We have changed this so that this line is displayed as header line in Katacoda

Reply:
I have changed it into header